### PR TITLE
[grpc] Update to v1.24.3

### PR DESCRIPTION
**Pull request by aliabbas2810**
Looking for zed sdk

### CODE REVIEW

**Review by JackBoosY — APPROVED** [review]

File: ports/grpc/portfile.cmake [hunk 1]
```diff
@@ -13,18 +13,16 @@ endif()
...
     OUT_SOURCE_PATH SOURCE_PATH
     REPO grpc/grpc
-    REF v1.23.1
-    SHA512 c39a07554645402c36cb79eddfda7165dc26ce5f7f09cae7ee6f7bedcc2aca8873117ea401d40ce2e3246e3e5888bcce12d4746cba7ed75068ff145046754981
+    REF v1.24.3
+    SHA512 19afa8b1fd64c429317c85d8cf42d01af0ac6aec82508740c176e1fd4f92dea9531ea9812523215ed67466bae93a75e14dc34cd9cce5987a13c0104e0313c69c
     HEAD_REF master
     PATCHES
         00001-fix-uwp.patch
-        00002-static-linking-in-linux.patch
```
Comment (JackBoosY) [inline ports/grpc/portfile.cmake]: Why modifying the order of these patches? It seems that the patch name is incorrect.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Comment by jozefizso** [comment]
This looks like a copy of the #8791

**Comment by JackBoosY** [comment]
Since #9135 contains a better fix, close this PR.

**Pull request closed without merging**